src/librpmem/rpmem_fip.c

```diff
@@ -206,19 +202,19 @@ rpmem_fip_set_closing(struct rpmem_fip *fip)
 }
 
 /*
- * rpmem_fip_lane_begin -- (internal) intialize list of events for lane
+ * lane_begin -- (internal) intialize list of events for lane
  */
 static inline void
-rpmem_fip_lane_begin(struct rpmem_fip_lane *lanep, uint64_t event)
+lane_begin(struct rpmem_fip_lane *lanep, uint64_t event)
 {
 	lanep->event = event;
 }
 
 /*
- * rpmem_fip_lane_init -- (internal) initialize single lane
+ * lane_init -- (internal) initialize single lane
  */
 static int
-rpmem_fip_lane_init(struct rpmem_fip *fip, struct rpmem_fip_lane *lanep)
+lane_init(struct rpmem_fip *fip, struct rpmem_fip_lane *lanep)
```

Comment:
We already have lane_init in another module, so I'd prefer this function to have a distinct name. The same applies to other lane_ functions.